src/main/java/mate/academy/bookstore/repository/shoppingcart/ShoppingCartRepository.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.bookstore.repository.shoppingcart;
+
+import java.util.Optional;
+import mate.academy.bookstore.model.ShoppingCart;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
+    @EntityGraph(attributePaths = {"cartItems", "cartItems.book"})
+    Optional<ShoppingCart> findByUserId(Long userId);
```

Comment:
```suggestion
    Optional<ShoppingCart> findByUserIdFetchCartItemsWithBooks(Long userId);
```

Reply:
I accidentally accepted changes here on GitHub and everything was reverted. With method name findByUserIdFetchCartItemsWithBooks, I got next error: No property 'fetchCartItemsWithBooks' found for type 'Long'